backend/controllers/auth.js

```diff
@@ -1,7 +1,8 @@
 import { HttpError } from "../helpers/HttpError.js";
 import { prisma } from "../lib/db.js";
+import jwt from "jsonwebtoken";
 
-const COOKIE_MAX_AGE_MILISECONDS = 60 * 60 * 1000;
+const { JWT_SECRET } = process.env;
```

Comment:
It seems you forgot to change the env.js validation file to parse JWT_SECRET. Please update it, and use the env.js file to import the environment. Parsing required values makes it possible to quickly identify the issue when the env variable is not yet set.